src/core/components/init-assets.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const path = require('path')
+const fs = require('fs')
+const glob = require('glob')
+const importer = require('ipfs-unixfs-engine').importer
+const pull = require('pull-stream')
+const file = require('pull-file')
+const mh = require('multihashes')
+
+// Add the default assets to the repo.
+module.exports = function addDefaultAssets (self, log, callback) {
+  const initDocsPath = path.join(__dirname, '../../init-files/init-docs')
+  const index = __dirname.lastIndexOf('/')
+
+  pull(
+    pull.values([initDocsPath]),
+    pull.asyncMap((val, cb) => {
+      glob(path.join(val, '/**/*'), cb)
+    }),
+    pull.flatten(),
+    pull.map((element) => {
+      const addPath = element.substring(index + 1, element.length)
+      if (fs.statSync(element).isDirectory()) return
+
+      return {
+        path: addPath,
+        content: file(element)
+      }
+    }),
+    pull.filter(Boolean),
+    importer(self._ipldResolver),
+    pull.through((el) => {
+      if (el.path === 'files/init-docs/docs') {
+        const hash = mh.toB58String(el.multihash)
+        log('to get started, enter:')
+        log()
+        log(`\t jsipfs files cat /ipfs/${hash}/readme`)
```

Comment:
Let's point directly to the readme hash, since we don't do traversals right now //cc @VictorBjelkholm